app/models/comment.rb

```diff
@@ -0,0 +1,11 @@
+class Comment < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+  after_save :update_comments_counter
+
+  private
+
+  def update_comments_counter
+    post.update(comments_counter: post.comments.length)
+  end
```

Comment:
- [ ] [OPTIONAL] Basically this method is supposed to increment the counter for the comments on a post for a specific user. This can easily be achieved by making use of the ActiveRecord increment!. Kindly consider using this. Have a look at [this](https://www.rubydoc.info/docs/rails/ActiveRecord%2FPersistence:increment!) or [this](https://stackoverflow.com/questions/11499110/increment-vs-1) for inspiration. 

**You will see how shorter this line will become** :smile:

Reply:
Hey @Nemwel-Boniface Done ✔️